modules/calc/src/main/java/com/opengamma/strata/calc/runner/CompositeCalculationFunctions.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.calc.runner;
+
+import java.util.Optional;
+
+import com.opengamma.strata.basics.CalculationTarget;
+import com.opengamma.strata.calc.runner.function.CalculationFunction;
+import com.opengamma.strata.collect.ArgChecker;
+
+/**
+ * A set of calculation functions which combines the functions in two other sets of functions.
+ */
+class CompositeCalculationFunctions implements CalculationFunctions {
```

Comment:
DefaultCalculationFunctions is a Joda-Bean "light" so it gets equals and hashCode and toString. Think this should be too.